### PR TITLE
remove references to new code base and cleanup

### PR DESCRIPTION
**Pull request by eneufeld**

### CODE REVIEW

File: content/documentation/coffeeEditorNg/content.md [hunk 1]
```diff
@@ -23,9 +23,13 @@ In order to also simplify the editing capabilities, including undo and redo, on
 
 Finally, the Coffee Editor NG language contribution adds a custom coffee-model-specific API to be used by clients in order to provide reusable model queries and manipulation functions to the clients.
 
+<!--  
 ➡️ TODO Link to try now
 
 ➡️ TODO Link to source code and a good entry point to look at the language contribution
+-->
+
+We will provide a live demo as well as a link to GitHub as soon the code is published.
```
Comment (CamilleLetavernier) [inline content/documentation/coffeeEditorNg/content.md]:
```suggestion
We will provide a live demo as well as a link to GitHub as soon as the code is published.
```

File: content/documentation/gettingStarted/content.md [hunk 2]
```diff
@@ -6,8 +6,13 @@ weight = 120
...
 TODO:
 * Link for trying it out online
 * Instructions on how to build, run and debug locally
 
-➡️ Now as you are up and running, let's have a look at the [Coffee Editor NG architecture]({{< relref  "coffeeeditorng" >}})!
\ No newline at end of file
+➡️ Now as you are up and running, let's have a look at the [Coffee Editor NG architecture]({{< relref  "coffeeeditorng" >}})!
+-->
+
+We will provide a live demo as well as a link to GitHub as soon the code is published.
```
Comment (CamilleLetavernier) [inline content/documentation/gettingStarted/content.md]:
```suggestion
We will provide a live demo as well as a link to GitHub as soon as the code is published.
```

**Review by CamilleLetavernier — COMMENTED** [review]
Thanks! Changes look good to me; just a couple of typos

File: content/_index/coffeeeditorlinks/tryitnow.md [hunk 3]
```diff
@@ -7,4 +7,4 @@ weight = 10
   url = "https://eclipsesource.com/coffee-editor"
 +++
 
-If you want to see the coffee editor in action, [launch a demo instance](https://eclipsesource.com/coffee-editor) of the coffee editor. This allows you to try any feature of the example. Please note that the demo will automatically shut down after 30 minutes and no data will be persisted.
+If you want to see the coffee editor in action, [launch a demo instance (not NG)](https://eclipsesource.com/coffee-editor) of the coffee editor. This allows you to try any feature of the example. Please note that the demo will automatically shut down after 30 minutes and no data will be persisted.
```
Comment (planger) [inline content/_index/coffeeeditorlinks/tryitnow.md]:
```suggestion
If you want to see the coffee editor in action, [launch a demo instance](https://eclipsesource.com/coffee-editor) of the EMF-based coffee editor. This allows you to try any feature of the example. Please note that the demo will automatically shut down after 30 minutes and no data will be persisted.
```

**Review by planger — APPROVED** [review]

**Pull request merged**